_quarto.yml

```diff
@@ -108,20 +113,11 @@ website:
           - RAP/rap-support.qmd
           - RAP/rap-faq.qmd
           - RAP/rap-statistics.qmd
-      - section: "ADA and Databricks"
+      - section: "Databricks Setup Guides"
         contents:
-          - text: "---"
-          - text: "Understanding Databricks"
-          - text: "---"
-          - ADA/ada.qmd
-          - ADA/databricks_fundamentals.qmd
-          - ADA/databricks_notebooks.qmd
-          - ADA/databricks_workflows.qmd
-          - text: "---"
-          - text: "How to..."
-          - text: "---"
           - ADA/databricks_rstudio_sql_warehouse.qmd
           - ADA/databricks_rstudio_personal_cluster.qmd
+          - ADA/databricks_rstudio_personal_cluster_sparklyr.qmd
```

Comment:
Should the new sparklyr page be combined into the same page as odbc / DBI and just be 'Personal cluster (R Studio)'? Seems like we're duplicating our own content here, which makes it harder for me to navigate and also means we'll need to keep on top of two versions of the same paragraphs?
![image](https://github.com/user-attachments/assets/6f169466-c18e-4632-aae0-3c9ca692a8b5)

Reply:
Fair enough, I definitely find it more confusing this way around (I'd find them easier in a single page), but happy to leave as is for now and you to look at at a later point